plugin/coc-powershell.vim

```diff
@@ -9,6 +9,7 @@ if(!exists("g:pses_logs_dir"))
     let g:pses_logs_dir      = resolve(expand(s:vimscript_dir . '/../.pses/logs/' . strftime('%Y%m%d') . '-' . getpid()))
 endif
 
+
```

Comment:
nit: delete extra line

Reply:
😂 I think this was an accident... I think I had some code in here that actually belonged in the autoload 🤷‍♂️